components/Tab/index.css

```diff
@@ -0,0 +1,51 @@
+.tabset {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.tabset__tab__handle {
+  min-width: 24px;
+  padding: 16px;
+  border-bottom: 2px solid transparent;
+  cursor: pointer;
+  text-decoration: none;
+  color: var(--black);
+}
+
+.tabset__tab__handle:hover {
+  background-color: var(--grey-90-a10);
+  border-bottom: 2px solid var(--grey-90-a20);
+  text-decoration: none;
+}
+
+.tabset__tab__handle:active {
+  background-color: var(--grey-90-a20);
+  border-bottom: 2px solid var(--grey-90-a20);
+}
+
+.tabset__tab--selected__handle {
+  color: var(--blue-60);
+  border-bottom: 2px solid var(--blue-50);
+}
+
+.tabset__tab--selected__handle:hover {
+  background-color: var(--grey-90-a10);
+  border-bottom: 2px solid var(--blue-50);
+}
+
+.tabset__tab--selected__handle:active {
+  background-color: var(--grey-90-a20);
+}
+
+.tabset__tab__handle::-moz-focusring {
+  outline: none;
+}
+
+.tabset__tab__handle:focus {
```

Comment:
if you make this `:-moz-focusring` and combine with the style above, instead of `:focus` it'll remove the active/pressed focus state

Reply:
though it won't work in other browsers...

Reply:
:focus doesn't have the same issues on chrome nor safari at least, can we detect if they are using firefox and in that case use :-moz-focusring?

Reply:
Not sure we can do any detection from just CSS - we'd like this stuff to work with just HTML & CSS if possible, too. We're making React components, but also trying to keep those mostly optional